files/migrations/0009_backfill_filename_fields.py

```
@@ -0,0 +1,129 @@
+                batch = []
+
+    # Update remaining records
+    if batch:
+        with transaction.atomic():
+            Encoding.objects.bulk_update(batch, ['filename'], batch_size=batch_size)
+        updated_count += len(batch)
+
+    print(f"Successfully backfilled filename for {updated_count} Encoding records.")
+
+
+def reverse_backfill(apps, schema_editor):
```

Ruff (0.14.4)

Unused function argument: `schema_editor`

(ARG001)